guides/common/modules/proc_configuring-for-uefi-http-boot-ipv6-provisioning.adoc

```diff
@@ -24,6 +24,9 @@ ifndef::foreman-deb[]
 # {package-update-project} grub2-efi
 ----
 endif::[]
-ifdef::katello,satellite,orcharhino[]
+ifdef::satellite[]
 . Synchronize the {RHEL} 8 kickstart repository.
 endif::[]
+ifdef::katello,orcharhino[]
+. Synchronize the EL 8 kickstart repository.
```

Comment:
Just a thought: would an `{EL}` macro be useful? That's `EL` in upstream, `{RHEL}` in Satellite?

Reply:
Yes, that would be lovely. It would also allow us to build guides for Alma Linux 8, RHEL8, OL8, Rocky Linux 8, and maybe even centos 8 stream.

I'll put it on my todo for later.